weldx/welding/groove/iso_9692_1.py

```diff
@@ -163,6 +211,16 @@ def to_profile(self, width_default: pint.Quantity = Q_(5, "mm")) -> geo.Profile:
 
         return geo.Profile([shape, shape_r], units=_DEFAULT_LEN_UNIT)
 
+    @property
+    def cross_sect_area(self):
+        from sympy.geometry import Polygon
+
+        b = float(self.b.magnitude)
+        t = float(self.t.magnitude)
```

Comment:
I guess there is no way to create a `Polygon` from `pint` quantities directly? That would be super convenient.
`pint` doesn't seem to work nicely with `Point2D`

Reply:
No, unfortunately the geometry classes of sympy dont play along with pint.